dockerfiles/theia/Dockerfile

```diff
@@ -82,6 +82,46 @@ RUN che:theia production
 # change permissions
 RUN find production -exec sh -c "chgrp 0 {}; chmod g+rwX {}" \; 2>log.txt
 
+
```

Comment:
hello, didn't see it before.

why do we have copy of lines 89--->115 from the previous container ?
couldn't we just add # Clone and build che-theia plugins at the end of previous container ?

here we're duplicating all github token, etc logic in the same file for almost nothing ?

Reply:
maybe i misunderstood the comments of https://github.com/eclipse/che-theia/pull/43#discussion_r252657776 
we are duplicating just the github token stuff. I am very fine with cloning and building che-theia plugins at the end of the previous container